src/patternfly/components/Select/select.scss

```diff
@@ -234,6 +234,12 @@
   .pf-c-select__toggle-clear + .pf-c-select__toggle-button {
     padding-left: var(--pf-c-select__toggle-button--PaddingLeft);
   }
+
+  .pf-c-select__toggle-arrow {
+    .pf-m-top.pf-m-expanded > & {
+      transform: var(--pf-c-select__toggle-arrow--m-top--m-expanded__toggle-arrow--Transform);
+    }
+  }
```

Comment:
nice! Can you remove this block - https://github.com/patternfly/patternfly-next/blob/e7fbc2dca3e3ad139ab85f66b65445cae66db587/src/patternfly/components/Select/select.scss#L252-L254?